### ISSUE TITLE
Jaws in browse mode does not announce tab panel boundries

### ISSUE THREAD

**Issue by Wildebrew** [FreedomScientific/standards-support#316]
### Summary

In order to distinguish the contents of a tab panel from other page content a screen reader user needs to know the borders of the panel.

This information is provided in markup by using `role="tabpanel` on the tab panel element, as well as `aria-labelledby` to tie its accessible name back to its associated tab.

```
<a href="#" id="tab1" role="tab" aria-selected="true">Tab 1</a>
..
<div role="tabpanel" aria-labelledby="tab1">
Tab panel content goes here
</div>
```

Screen readers should communicate the start and end of the tab panel in browse mode. For the code snippet above screen readers should:
* announce "tab 1 tab panel" or "tab1 tab panel start" at the beginning of the tab panel
* announce "tab 1 tab panel" or "tab 1 tab panel end" at the end of the tab panel.

## Example

Open [This tabs example from APG](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html)

Navigate past the tabs and into the tabpanel in browse mode (using arrow keys).

### Expected result

Before the sentence that starts with "Nils Frahm is " Jaws should say "Nils Frahm tabpanel" 

After the sentence that ends with "Moog Taurus." Jaws should say "tab panel end" or "Nils Frahm tab panel end".

### Actual result

Jaws + IE is the only combo that does this. Using Jaws 19 with Chrome or Firefox nothing is announced for the tab panel (tested at various user verbosity levels).

If the tab pane contains a focusable element, the tabpanel role and name are announced (in both Jaws and Firefox) but that is not sufficient since tab content does not have to contain focusable elements and announcing the tab panel in the context of focusable elements does not give a clear picture of the tab panel boundries.

#### JAWS version and build number

Jaws 2019.1812.49

#### Operating System and version
Windows 10

#### Browser and version:
Latest public versions of Chrome and Firefox, + IE11.

**Comment by JAWS-test** [comment]
I can confirm the incorrect output in Chrome and Firefox and have noticed the following additional problems (all on https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html)
* When reading with the arrow keys, the beginning and end of the tablist area are output as a "group" (with its label). 
* No role is output for tab navigation to the tab panel.
* If the role=tab receive the focus with Tab key, the role is output twice

These three problems do not occur with IE 11.

**Comment by janellenchew** [comment]
This is a duplicate of #151 .

**Comment by stevefaulkner** [comment]
Closing this, please feel free to re-open if this still occurs with Latest (JAWS 2024).

**Comment by JAWS-test** [comment]
The problem of missing tabpanel output no longer occurs with JAWS 2024

**Comment by stevefaulkner** [comment]
@JAWS-test thanks
